projects/rero/ng-core/src/lib/record/search/aggregation/buckets/buckets.component.ts

```diff
@@ -36,16 +36,44 @@ export class BucketsComponent implements OnInit, OnDestroy, OnChanges {
 
   /** More and less on aggregation content (facet) */
   moreMode = true;
-
-  /** Current selected values for the aggregation */
+  /** Current selected values for the aggregations */
   aggregationsFilters: Array<AggregationsFilter> = [];
-
   /** Children of current bucket */
   bucketChildren: any = {};
 
   /** Subscription to aggregationsFilters observable */
   private _aggregationsFiltersSubscription: Subscription;
 
+
+  // GETTERS & SETTERS =================================================================
+  /** Returns selected filters for the aggregation key. */
+  get aggregationFilters(): Array<string> {
+    const aggregationFilters = this.aggregationsFilters.find((item: AggregationsFilter) => item.key === this.aggregationKey);
+    return aggregationFilters === undefined
+        ? []
+        : aggregationFilters.values;
+  }
+
+  /** Get bucket size. */
+  get bucketSize(): number {
+    const size = this.buckets.length;
+    return this.size === null || this.moreMode === false
+        ? size
+        : this.size;
+  }
+
+  /** Get buckets with non-zero doc_count. */
```

Comment:
Is it get the buckets or a sum of the number of doc_count?

Reply:
it gets the number of bucket items with non-zero doc_count. I updated the docstring